test/integration/dnet/multi.bats

```diff
@@ -22,6 +22,7 @@ function is_network_exist() {
 	run dnet_cmd $(inst_id2port $i) network create -d test ${oname}
 	echo ${output}
 	[ "$status" -eq 0 ]
+	done
```

Comment:
Ah right that makes more sense. I'll have to do some debugging then as this test kept failing for some reason.